.github/workflows/npm-notification-publish.yml

```diff
@@ -0,0 +1,81 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Notification Package Release
+
+on:
+  workflow_dispatch:
+  push:
+      paths: 
+        - 'package/notification'
+      branches:
+        - 'main'
+
+permissions:
+  issues: read
+  pull-requests: read
+
+jobs:
+  build-notif-package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - run: cd packages/notification
+      - run: npm ci
+      - run: npm run build
```

Comment:
I think it should be removed. It's a redundant job, since it will be build again in `publish-npm` job